src/rabbit_mgmt_db_handler.erl

```diff
@@ -63,6 +70,7 @@ ensure_statistics_enabled() ->
             ok
     end,
     {ok, StatsLevel} = application:get_env(rabbit, collect_statistics),
+    rabbit_log:info("Using rates mode '~p'~n", [rates_mode()]),
```

Comment:
One last comment: a prefix to remind the context would be helpful in this message. Something like "Management plugin: using rates mode ...".